### ISSUE TITLE
seekToPlayer error 

### ISSUE THREAD

**Issue by zfha** [dooboolab/flutter_sound#70]
in iOS Platform
_TypeError (type 'int' is not a subtype of type 'String')

**Comment by hyochan** [comment]
Can you also try `example` project?

**Comment by nicholas-vella** [comment]
I've just opened a pull request for this !72

**Comment by CoreyCole** [comment]
I see this is merged, but is it live in `1.4.0`? I'm getting the error on iOS about the return type of seek:
```
[VERBOSE-2:ui_dart_state.cc(148)] Unhandled Exception: type 'int' is not a subtype of type 'String'
#0      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:302:13)
<asynchronous suspension>
#1      FlutterSound.seekToPlayer (package:flutter_sound/flutter_sound.dart:190:24)
<asynchronous suspension>
#2      AudioService.seekPlayer (package:mokri_storytime/src/blocs/services/audio_service.dart:219:19)
```
